tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
> I wonder why the bracket was moved.

I thought it was a mistake. It just looks weird - single bracket on the line.
Speaking of other methods, I believe there is quiet a difference between methods and closures. The last serve to shorten the code so it would be nice to format them as short as possible.